podcasts/SwitchCell.swift

```diff
@@ -46,4 +46,29 @@ class SwitchCell: ThemeableCell {
 
     override func setSelected(_ selected: Bool, animated: Bool) {}
     override func setHighlighted(_ highlighted: Bool, animated: Bool) {}
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        setupAccessibility()
+    }
+
+    override func accessibilityActivate() -> Bool {
+        cellSwitch.setOn(!cellSwitch.isOn, animated: true)
+        cellSwitch.sendActions(for: .valueChanged)
+        matchAccessibilityValueWithSwitchControl()
+        return true
+    }
+
+    private func setupAccessibility() {
+        isAccessibilityElement = true
```

Comment:
I tried setting isAccessibilityElement to true, but it didn't work as voiceover for some reason tries to read accessibility label from its parent view (this cell).